ci-operator/config/openshift/appliance/openshift-appliance-master.yaml

```diff
@@ -63,16 +63,7 @@ tests:
   steps:
     allow_skip_on_success: true
     cluster_profile: equinix-ocp-metal
-    dependencies:
-      APPLIANCE_IMAGE: agent-preinstall-image-builder
-    env:
-      AGENT_E2E_TEST_BOOT_MODE: DISKIMAGE
-      DEVSCRIPTS_CONFIG: |
-        AGENT_E2E_TEST_BOOT_MODE=DISKIMAGE
-        AGENT_E2E_TEST_SCENARIO=COMPACT_IPV4
-        NETWORK_TYPE=OVNKubernetes
-        INSTALLER_PROXY=true
-    workflow: agent-e2e-generic
+    workflow: agent-e2e-appliance-ipv4
```

Comment:
Shouldn't change the `cluster_profile` also here, for the all config file in this folder?

Reply:
:man_facepalming: yes